### PR TITLE
security policy edits for clarity

### PR DESCRIPTION
**Pull request by billwanjohi**
No actual changes in policy are being proposed here.

[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/security-policy-edits/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/03-policies/security.md
Comment (billwanjohi) [inline docs/03-policies/security.md:37]: Not obvious in the GitHub diff, but I just added a comma after `Generally`.

**Review by erogray — APPROVED** [review]

File: docs/03-policies/security.md
Comment (openprivacy) [inline docs/03-policies/security.md:21]: @erogray @billwanjohi This looks good to me, but I'd like to run this new wording by @marctjones as he provided the original legalese for defining confidential information.

Comment (openprivacy) [reply docs/03-policies/security.md:21]: @marctjones the rest of the changes in this file are fine, but please review the replacement of lines 21 & 22 above with the bulleted lines 21-26 below. I agree with the change, but as you provided the initial edit, I'd appreciate your blessing.

Comment (openprivacy) [reply docs/03-policies/security.md:21]: @marctjones I failed to submit my comments above before the merge was accepted. Here's a link to the (now live) agreement: https://handbook.civicactions.com/en/latest/03-policies/security/#confidential-information-agreement

I believe Bill's changes are good, but would like your review (as you defined "confidential information" on the last update).

File: docs/03-policies/security.md
Comment (marctjones) [reply docs/03-policies/security.md:37]: Thanks. Adding the comma makes sense

File: docs/03-policies/security.md
Comment (marctjones) [inline docs/03-policies/security.md:191]: We should keep the original sentence. Can we remove the change for this line so it reads "Software that is not licensed under an approved CivicActions open source license may not be used on a project without prior approval from the legal team."

File: docs/03-policies/security.md
Comment (marctjones) [inline docs/03-policies/security.md:204]: We don't have a CIO. We have a CTO. 

Instead of "routes", I think a better word would be "methods"

File: docs/03-policies/security.md
Comment (marctjones) [inline docs/03-policies/security.md:207]: Is "vhost" still the right word. Seems like "server" or "server instance" might be more accurate.

File: docs/03-policies/security.md
Comment (marctjones) [reply docs/03-policies/security.md:21]: This looks fine to me. Thanks for including me in the review @openprivacy . I would like to review any policies that have legal consequences.

**Pull request merged**
